source/agora/cli/checkvtable/generate.d

```diff
@@ -19,7 +19,10 @@ import std.file;
 import std.format;
 import std.stdio;
 
-version (Windows) { pragma(msg, "This does not support Windows."); } else:
+version (Windows)
+{
+    pragma(msg, "The agora-checkvtable-gen : This does not support Windows.");
```

Comment:
How about inserting the module name instead of the text of "The agora-checkvtable-gen"?